.github/workflows/aes.yml

```diff
@@ -71,6 +71,7 @@ jobs:
       - run: ${{ matrix.deps }}
       - run: cargo check --target ${{ matrix.target }} --all-features
       - run: cargo test --release --target ${{ matrix.target }}
+      - run: cargo test --release --target ${{ matrix.target }} --features autodetect
```

Comment:
We probably also should specifically test the AES-NI implementation by enabling the `aes` target feature via RUSTFLAGS.

Reply:
It does: https://github.com/RustCrypto/block-ciphers/blob/0fb7bb55fc851db10cade44de9eaf11b08441046/.github/workflows/aes.yml#L82-L85